insta/src/lib.rs

```diff
@@ -77,10 +77,10 @@
 //! This crate exports multiple macros for snapshot testing:
 //!
 //! - [`assert_snapshot!`] for comparing basic snapshots of
-//!   [`Display`](std::fmt::Display) outputs, often strings.
+//!   [`std::fmt::Display`] outputs, often strings.
```

Comment:
How does this appear in markdown? This is more user-focused docs, so I think we just want the `Display` to appear, as though we're speaking it out loud.

(most of the other changes are fine, that's more API docs)

Reply:
I think this appears as documented. At least for `crate::`, it does.
Lets revert this

```suggestion
//!   [`Display`](std::fmt::Display) outputs, often strings.
```